tests/pages/api/event/[id]/update.test.ts

```diff
@@ -19,6 +19,15 @@ describe("PATCH /api/event/:id", () => {
         getAuth: vi.fn().mockReturnValue({ userId: "Test" })
       };
     });
+    vi.mock("@clerk/nextjs", () => {
+      return {
+        clerkClient: {
+          users: {
+            getUserList: vi.fn().mockReturnValue(mockClerkUsers)
+          }
+        }
+      };
+    });
```

Comment:
This could be moved to a reusable/wrapper function in the utils file. If possible, could you move getAuth mock as well.